backend/src/db/migrations/20250331140313-commenter-observes-post.ts

```diff
@@ -0,0 +1,61 @@
+import { getDriver } from '../../db/neo4j'
+
+export const description = `
+All users commenting a post observe the post.
+`
+
+export async function up(next) {
+  const driver = getDriver()
+  const session = driver.session()
+  const transaction = session.beginTransaction()
+
+  try {
+    // Implement your migration here.
+    await transaction.run(`
+      MATCH (commenter:User)-[:COMMENTS]->(post:Post)
```

Comment:
```suggestion
      MATCH (commenter:User)-[:WROTE]->(:Comment)-[:COMMENTS]->(post:Post)
```

This reaches your goal.
Your doesn't work.

Cypher to query in the database:

`MATCH (author:User)-[:WROTE]->(comment:Comment)-[:COMMENTS]->(post:Post) RETURN author, post;`

Reply:
Thanks for finding this!